README.md

```diff
@@ -1432,15 +1430,15 @@ Sample usage :
 
 ```wdl
 task sparkjob_with_yarn_cluster {
-        .....
+        #.....
```

Comment:
I suspect because it's not syntactically correct but it's to demonstrate that one could have more stuff?